mission_control/navigator_missions/navigator_singleton/navigator.py

```diff
@@ -100,14 +126,34 @@ def _make_bounds(self):
 
     def vision_request(self, request_name, **kwargs):
         print "DEPREICATED: Please use new dictionary based system."
```

Comment:
May I suggest throwing [DeprecationWarning](https://docs.python.org/2/library/exceptions.html#exceptions.DeprecationWarning) in the future for increased legitness.